docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:14.1-alpine
+#    restart: always
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=pass
+    ports:
+      - '5432:5432'
```

Comment:
Change, please, mapping DB port 5432 to 5455, for example. As another container already uses this port.

Reply:
Done